package-lock.json

```diff
@@ -1372,7 +1372,7 @@
     "base64-js": {
       "version": "1.2.1",
       "resolved": "https://registry.npmjs.org/base64-js/-/base64-js-1.2.1.tgz",
-      "integrity": "sha512-dwVUVIXsBZXwTuwnXI9RK8sBmgq09NDHzyR9SAph9eqk76gKK2JSQmZARC2zRC81JC2QTtxD0ARU5qTS25gIGw==",
+      "integrity": "sha1-qRlH2h9KUW6jjltOwOw3c2deCIY=",
```

Comment:
This hash change thing is really annoying when working with `npm5` :(

Reply:
Yes it is, and we should not be introducing these changes here. I can usually resolve the issue by resetting `package-lock.json` to the original version, `rm -rf node_modules`, then a fresh `npm install`.